src/piadm.sh

```diff
@@ -624,20 +626,17 @@ remove() {
 
 ispoolenabled() {
 	pool=$1
-	poolpresent $pool
+	poolpresent "$pool"
 
 	# SmartOS convention is $POOL/boot.
-	currbootfs=$(zpool list -Ho bootfs $pool)
-	if [[ "$currbootfs" == "${pool}/boot" ]]; then
-		output=$(zfs list -H $currbootfs 2>&1)
-		if [[ $? -eq 0 ]]; then
-			# We're bootable (at least bootable enough)
-			return 0
-		fi
+	currbootfs=$(zpool list -Ho bootfs "$pool")
+	if [[ $currbootfs == "${pool}/boot" ]]; then
+		# We're bootable (at least bootable enough)
+		zfs list -H "$currbootfs" 2>&1  && return 0
```

Comment:
Hmmm, old development had used $output.  Your change, which correctly understand "output" should no longer be used, does need to silence the output.  Perhaps:

`zfs list -H "$currbootfs" > /dev/null 2>&1 && return 0` 

?

Reply:
fixed.